pom.xml

```diff
@@ -2,13 +2,19 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.neo4j</groupId>
     <artifactId>neo4j-jdbc</artifactId>
-    <version>2.3-SNAPSHOT</version>
+    <version>2.0.5-SNAPSHOT</version>
```

Comment:
There were actually, at leasts in http://m2.neo4j.org/content/repositories/releases.
Releasing `2.0.5` would mean (this works with Neo4j 2.0.5 onwards).

But it's confusing... because it's as feature-complete as version `2.3.2`...
I'm not 100% sure what the best versioning scheme would be in this case...